bears/js/JSHintBear.py

```diff
@@ -209,6 +209,14 @@ def generate_config(filename, file,
         :param use_mozilla_extension:
             This options tells JSHint that your code uses Mozilla JavaScript
             extensions.
+        :param javascript_strictness:
+            Determines what sort of strictness to use in the JavaScript code.
+            The possible options are:
+
+            - "global" - there must be a ``"use strict";`` at global level
+            - "implied" - lint the code as if there is a ``"use strict";``
+            - False - disable warnings about strict mode
+            - True - there must be a ``"use strict";`` at function level
```

Comment:
with this you're suggesting to the user that he should write `"global"` including the quotes as true and false are quoteless which isn't what we want right?